scalar/inet.go

```diff
@@ -129,6 +129,12 @@ func (s *Inet) Set(val any) error {
 	return nil
 }
 
+func (s *Inet) ByteSize() int64 { return int64(len(s.Value.IP) + len(s.Value.Mask)) }
```

Comment:
I'm not sure here how these get serialized so feel free to resolve this if not applicable, but do we need to track the `/` separator as well?

Reply:
we are looking only at the bytes being sent, so there's no real reason to account for the separator when it'll be tracked on the destination side once it decides on the destination value